librariesio-url-parser.gemspec

```diff
@@ -0,0 +1,21 @@
+# coding: utf-8
+lib = File.expand_path('../lib', __FILE__)
+$LOAD_PATH.unshift(lib) unless $LOAD_PATH.include?(lib)
+require "librariesio-url-parser"
+
+Gem::Specification.new do |spec|
+  spec.name          = "librariesio-url-parser"
+  spec.version       = LibrariesioURLParser::VERSION
+  spec.authors       = ["Andrew Nesbitt"]
+  spec.email         = ["andrewnez@gmail.com"]
+
+  spec.summary       = "Parse URLs"
+  spec.homepage      = "https://github.com/librariesio/librariesio-url-parser"
+  spec.license       = "AGPL-3.0"
```

Comment:
One thing I've been trying to add to older gems is this, which requires owners on rubygems.org to have MFA enabled:

`spec.metadata = { "rubygems_mfa_required" => "true" }`